test/memory.test.js

```diff
@@ -201,6 +201,7 @@ describe('Memory connector', function() {
       birthday: {type: Date, index: true},
       role: {type: String, index: true},
       order: {type: Number, index: true, sort: true},
+      notes: {type: String, index: true},
```

Comment:
ditto, I think the property should be `tag` not `notes`

Reply:
Yes you are right! fixed!